lib/active_model/serializer/caching.rb

```diff
@@ -145,6 +154,172 @@ def fragment_cache_enabled?
           perform_caching? && cache_store &&
             (_cache_only && !_cache_except || !_cache_only && _cache_except)
         end
+
+        # Read cache from cache_store
+        # @return [Hash]
+        def cache_read_multi(collection_serializer, adapter_instance, include_tree)
+          return {} if ActiveModelSerializers.config.cache_store.blank?
+
+          keys = object_cache_keys(collection_serializer, adapter_instance, include_tree)
+
+          return {} if keys.blank?
+
+          ActiveModelSerializers.config.cache_store.read_multi(*keys)
+        end
+
+        # find all cache_key for the collection_serializer
+        # @param collection_serializer
+        # @param include_tree
+        # @return [Array] all cache_key of collection_serializer
+        def object_cache_keys(collection_serializer, adapter_instance, include_tree)
+          cache_keys = []
+
```

Comment:
look at e.g. https://github.com/rails-api/active_model_serializers/pull/1276 in 0.8 for how we expand cache keys